compiler/src/scala/quoted/runtime/impl/printers/SourceCode.scala

```diff
@@ -126,6 +126,12 @@ object SourceCode {
         this += "."
         printImportSelectors(selectors)
 
+      case Export(expr, selectors) =>
+        this += "export "
+        printTree(expr)
+        this += "."
+        printImportSelectors(selectors)
+
```

Comment:
`ImportSelector` should be renamed to `Selector`

Reply:
done